src/App.tsx

```diff
@@ -1,6 +1,29 @@
+import {BrowserRouter as Router, Routes, Route, Link} from 'react-router-dom'
+
 function App() {
   return (
-    <div className="">hello sparkeats</div>
+    <Router>
+      <div className="">hello sparkeats</div>
+      <nav>
```

Comment:
A `nav` is unnecessary at this point since it isn't in the original design. See https://eats.seesparkbox.com/refresh.